### PR TITLE
adding possibility to show user description in sidebar

### PR DESCRIPTION
**Pull request by andrewkikot**

### CODE REVIEW

File: src/app/layouts/sidebar/sidebar.component.html [hunk 1]
```diff
@@ -23,6 +23,11 @@
       <a data-toggle="collapse" href="#collapseExample" class="collapsed">
         <span class="user-name">
             <em class="truncated pr-4">{{principal.getName()}}</em>
+            <em class="user-description text-muted truncated pr-4"
+                *ngIf="(userDescription$ | async)"
```
Comment (major13ua) [inline src/app/layouts/sidebar/sidebar.component.html]: you can use only 1 async - (userDescription$ | async as varName) and then just use varName

File: src/app/layouts/sidebar/sidebar.component.ts [hunk 2]
```diff
@@ -399,6 +409,22 @@ export class SidebarComponent implements OnInit, OnDestroy, AfterViewInit {
         }
     }
 
+    private getUserDescription(config: any): Observable<any> {
```
Comment (major13ua) [inline src/app/layouts/sidebar/sidebar.component.ts]: ctachError and return {} in any error, to prevent async close

Comment (andrewkikot) [reply src/app/layouts/sidebar/sidebar.component.ts]: +

File: src/app/layouts/sidebar/sidebar.component.html (hunk 1 above)
Comment (andrewkikot) [reply src/app/layouts/sidebar/sidebar.component.html]: +

**Pull request merged**